DogeNet.BLL/Extentions/FluentValidationExtentions.cs

```diff
@@ -13,5 +13,11 @@ public static class FluentValidationExtentions
             var type = typeof(T);
             return rule.WithMessage("{type} entity is not found");
         }
+
+        public static IRuleBuilderOptions<T, TProperty> ImgUrlValid<T, TProperty>(this IRuleBuilderOptions<T, TProperty> rule)
```

Comment:
This update should be implemented in scope of other PR?

Reply:
Yes, I will delete this from this pull request (it got 0 references I think I forgot to change branch some time ago)